src/java/org/apache/cassandra/tools/BulkLoadConnectionFactory.java

```diff
@@ -70,4 +70,9 @@ private NettyStreamingChannel create(OutboundConnectionSettings template, int me
 
```

Comment:
The comment in the second `create()` method is incorrect now. The preferred address is always ignored, not just ignored when encryption is enabled. The only reason that overload exists is so we don't call the `NettyStreamingConnectionFactory` implementation.